script/jenkins-utilities

```diff
@@ -166,7 +166,8 @@ function set_esp_remote_rollout_info_env() {
 
   case "${target_env}" in
     'gce' )
-      endpoint_status=`gcloud compute ssh "${INSTANCE_NAME}" --command="curl \"$endpoint_status_url\""`;;
+      set_ssh_private_key
+      endpoint_status=`${GCLOUD} compute ssh "${INSTANCE_NAME}" --command="curl \"$endpoint_status_url\""`;;
```

Comment:
Do we need to use "ssh" directly as 

https://github.com/cloudendpoints/esp/blob/master/script/jenkins-utilities#L343

Reply:
No, the caller of `save_vm_logs` is from a GCE VM, where gcloud is not avaialble.
https://github.com/cloudendpoints/esp/blob/master/test/bookstore/gce/e2e.sh#L131